landscape.yml

```diff
@@ -820,6 +820,15 @@ landscape:
               artwork_url: https://github.com/Normation/rudder/tree/master/logo
               blog_url: https://www.rudder.io/blog/
               youtube_url: https://www.youtube.com/user/RudderProject
+          - item:
+            name: sigstore
+            homepage_url: https://www.sigstore.dev/
+            logo: sigstore.svg
+            crunchbase: https://www.crunchbase.com/organization/sigstore
```

Comment:
this should probably point to the LF or OpenSSF

Reply:
Thanks, @caniszczyk , but I'm unclear. Which should point to LF or OpenSSF?